src/Microsoft.AspNetCore.Mvc.ViewFeatures/Internal/SaveTempDataPropertyFilter.cs

```diff
@@ -26,6 +27,10 @@ public SaveTempDataPropertyFilter(ITempDataDictionaryFactory factory)
 
         public IDictionary<PropertyInfo, object> OriginalValues { get; set; }
 
+        /// <summary>
+        /// Puts the modified values of <see cref="Subject"/> into <paramref name="tempData"/>.
+        /// </summary>
+        /// <param name="tempData">The <see cref="ITempDataDictionary"/> to be updated.</param>
```

Comment:
Super unrelated to your PR, but we should be using `PropertyHelpers` to do the `GetValue`. Right now it's doing a reflection call.